frontend/src/components/common/info/messages/EmptySessionsTabMessage.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { type ReactElement } from "react";
+import { Center, useStyleConfig } from "@chakra-ui/react";
+
+import DistributeAssessmentsIllustration from "../../../../assets/illustrations/distribute-assessments.svg";
+import type { TestSessionStatus } from "../../../../types/TestSessionTypes";
+import MessageContainer from "../MessageContainer";
+
+const BODY_PARAGRAPHS_BY_STATUS: Record<TestSessionStatus, string[]> = {
+  UPCOMING: ["Create an assessment to view it here."],
+  ACTIVE: [
+    "Upcoming assessments will appear here once they are open for students.",
+  ],
+  PAST: [
+    "Active assessments will appear here once they are no longer open for students.",
```

Comment:
```suggestion
    "Assessments will appear here once they are no longer open for students.",
```
can see how this makes sense, but is a little confusing imo